vm/src/stdlib/sre.rs

```diff
@@ -28,6 +28,12 @@ mod _sre {
     #[pyattr]
     pub use sre_engine::{constants::SRE_MAGIC as MAGIC, CODESIZE, MAXGROUPS, MAXREPEAT};
 
+    #[pyattr]
+    fn copyright(_vm: &VirtualMachine) -> &str {
+        let copy_right = " SRE 2.2.2 Copyright (c) 1997-2002 by Secret Labs AB ";
+        copy_right
+    }
+
```

Comment:
```suggestion
    #[pyattr(name = "copyright")]
    const COPYRIGHT : &str = " SRE 2.2.2 Copyright (c) 1997-2002 by Secret Labs AB ";
```

Reply:
Oh, I missed that... But It seems also true because most of the things offered by `sre.rs` and `_sre.c` are the same.

Reply:
That means the API is the same. It doesn't necessarily mean the code is directly translation from one source to the other one. Though we now have unclear points due to google vs oracle case, basically copyright came from copy of code, not from copy of signatures.